### PR TITLE
Added exception page

### PR DESCRIPTION
**Pull request by cmondragon15117**
This page is where the user is redirected upon any exception.

Was tested by dividing by zero, this code was removed to avoid any real exceptions.

-------------------------------------------------
Not sure if I should add tests to check if this is working
Next step is to add the redirected 404 page

### CODE REVIEW

**Comment by clague17** [comment]
So you're still working on this PR? You'll be adding more commits?

**Comment by cmondragon15117** [comment]
Yes, I just need to test the second error page which will handle a 404 error. I'll be adding that commit later today.

File: src/main/webapp/WEB-INF/view/error.jsp [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <% if(request.getSession().getAttribute("user") != null){ %>
+      <a href="/logout">Logout</a>
+    <% } %>
+  </nav>
+
+  <h1 id="container">
+    <div style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
+      Oops! An error happened...<br/>
+      Help report this error by raising an issue in our GitHub page.<br/>
+      <a href="https://github.com/Gonzo717/CodeU-Summer-2018">GitHub Link</a><br/>
```
Comment (Gonzo717) [inline src/main/webapp/WEB-INF/view/error.jsp]: This is so cool! Thank you for this!

Comment (cmondragon15117) [reply src/main/webapp/WEB-INF/view/error.jsp]: Thanks! And it automatically redirects to the error-page when there's an exception.

**Comment by cmondragon15117** [comment]
So, I was reading about redirection for the 404 error, but it seems like most web browsers will display their friendly error page, and it's currently unavailable to turn off for chrome. So I think I will just merge this for now.

**Pull request merged**